src/components/Nav/Components/NavCategoriesIcon.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import * as S from '../Nav.styles';
+
+export default function NavCategoriesIcon({ icon, text }) {
+  return (
+    <S.NavCategoriesIcon>
+      <div>{icon}</div>
+      <p>{text}</p>
+    </S.NavCategoriesIcon>
```

Comment:
scss를 사용할 때는 가장 부모태그의 className과 컴포넌트의 이름을 동일하게 하라고 말씀드렸지만
스타일드 컴포넌트에서는 컴포넌트 안에 있는 각 태그들도 사실상 컴포넌트기 때문에 이름이 겹쳐서 에러가 발생할 수 있습니다.

아래와 같은 방식으로 컴포넌트를 구현해주세요

```suggestion
export default function NavCategoriesIcon({ icon, text }) {
  return (
    <S.IconContainer>
      <div>{icon}</div>
      <p>{text}</p>
    </S.IconContainer>
```